src/tokens/transformation/tailwind/extractComponentStyles.js

```diff
@@ -0,0 +1,101 @@
+const svelte = require('svelte/compiler')
+const { readdirSync, statSync } = require('fs')
+const { writeFile, readFile } = require('fs/promises')
+const { join, parse } = require('path')
+const preprocess = require('svelte-preprocess')
+const postcssJs = require('postcss-js')
+const postcss = require('postcss')
+const sortMediaQueries = require('postcss-sort-media-queries')()
+const theme = require('../../../postcss/theme')
+const { walk } = require('../../../scripts/common')
+
+const tokenCategories = [
+  'font',
+  'typography',
+  'color',
+  'spacing',
+  'radius',
+  'effect',
+  'gradient'
+]
+
+const leoVarsPattern = RegExp(
```

Comment:
It was originally there in a larger attempt to convert Leo variables in the component CSS to use Tailwind `theme` functions so as to avoid loading the `variables.css` file. I ran into a snag, though, so this could be a future improvement, but not now.